playbooks/common/openshift-checks/health.yml

```diff
@@ -11,6 +8,6 @@
   vars:
   - r_openshift_health_checker_playbook_context: "health"
   post_tasks:
-  - action: openshift_health_check  # https://github.com/ansible/ansible/issues/20513
+  - action: openshift_health_check
```

Comment:
Seemed bcoca was saying it would work if there were a module file for the action plugin. Not sure. Anyway this is fine.